app/classes/api2/api_key_api.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
 class API2
-  # API for ApiKey
-  class ApiKeyAPI < ModelAPI
-    self.model = ApiKey
+  # API for APIKey
+  class APIKeyAPI < ModelAPI
+    self.model = APIKey
 
```

Comment:
Haha, that's *awesome*!  And I apparently created the name "ApiKeyAPI" with a completely straight face. :)